libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -74,4 +81,49 @@ export class StripeManager {
       customer.tax?.automatic_tax === 'not_collecting'
     );
   }
+
+  /**
+   * Updates customer object with incoming tax ID if existing tax ID does not match
+   *
+   * @param customerId Customer ID of customer to be updated
+   * @param taxId Customer tax ID to be updated if different from existing tax ID
+   * @returns Updated customer, or undefined if existing customer tax ID matches
+   */
+  async setCustomerTaxId(customerId: string, taxId: string) {
+    const customerTaxId = await this.getCustomerTaxId(
+      customerId,
+      MOZILLA_TAX_ID
+    );
+
+    if (customerTaxId !== taxId || customerTaxId === '') {
```

Comment:
I think a safer check here would be `if (customerTaxId !== taxId || !customerTaxId) {`

Reply:
Done!